check_disk_btrfs

```diff
@@ -173,27 +187,50 @@ def find_hr_bytes(total, used):
 
     return hr_label, hr_total, hr_used
 
-def main():
+def cli(args):
+    """
+    Defines the CLI Parser and parses the arguments
+    """
     arg_parser = ArgumentParser(description= "check_disk_btrfs (Version: %s)" % (VERSION))
-    arg_parser.add_argument('-S', '--sudo', help="use sudo", default=1)
-    arg_parser.add_argument('-t', '--timeout', type=int, help="plugin timeout", default=30)
-    arg_parser.add_argument('-U', '--unallocated', help="consider unallocated blocks by using overall size as total", default=1)
-    arg_parser.add_argument('-w', '--threshold-warning', type=int, default=80, help="warning threshold in percent")
-    arg_parser.add_argument('-c', '--threshold-critical', type=int, default=90, help="critical threshold in percent")
-    arg_parser.add_argument('-V', '--volume', default='/', help="btrfs volume") #TODO support multiple volumes
-    arg_parser.add_argument("-v", "--verbose", action="store_true", help="increase output verbosity")
-    arg_parser.add_argument("--btrfs_path", default=os.environ.get("BTRFS", "/usr/sbin/btrfs"), help="Specify the btrfs path to the executable")
-    arg_parser.add_argument("--sudo_path", default=os.environ.get("SUDO", "/usr/bin/sudo"), help="Specify the sudo path to the executable")
-    arg_parser.add_argument("-m", "--missing", action="store_true", help="check for missing devices in raid array")
-    arg_parser.add_argument("-e", "--error", action="store_true", help="check for scrub errors in device")
-    args = arg_parser.parse_args(sys.argv[1:])
-
-    global verbose, sudo_path, btrfs_path
-    verbose = args.verbose
-    sudo_path = args.sudo_path
-    btrfs_path = args.btrfs_path
-
-    output = check_usage(args.volume, args.timeout, args.sudo)
+    arg_parser.add_argument('-S', '--sudo', type=int, default=1,
```

Comment:
Why not use action="store_true" here ?

Reply:
Great question, I was thinking the same... however. If we store_true and have the default True (which is currently the default with default=1) then we can never set it to False. 

Personally I think it would be best to remove the --sudo flag entirely and just use a sudo rule for the check plugin. But I didn't want to do this major change yet.